src/confluent_kafka/schema_registry/schema_registry_client.py

```diff
@@ -372,26 +545,41 @@ class SchemaRegistryClient(object):
         `Confluent Schema Registry documentation <http://confluent.io/docs/current/schema-registry/docs/intro.html>`_
     """  # noqa: E501
 
-    def __init__(self, conf):
+    def __init__(self, conf: dict):
+        self._conf = conf
         self._rest_client = _RestClient(conf)
         self._cache = _SchemaCache()
```

Comment:
Is there any case where we would want TTL for schemaCache and RegisteredSchemaCache?

Reply:
Not in normal scenarios